### DISCUSSION TITLE
Anomaly in calculating diffusive conductance

### DISCUSSION THREAD

**Discussion by kunal0kumar0sinha** [PMEAL/OpenPNM#2724] [Development]
how to control throat length. I have kept spacing and pore diameter constant, thenever throat length is not constant. 

![image](https://user-images.githubusercontent.com/123363487/229711435-f249e86f-d29a-4f8a-aceb-16b022f01b8d.png)

**Comment by likeduck** [comment]
There is a dependency graph in openpnm, `throat.length` has many influence parameters, you can try to print them to check.
![Figure_1](https://user-images.githubusercontent.com/117707506/229745864-d5e99c1a-081c-4cc8-915c-e2a7ab05020d.png)
